.pre-commit-config.yaml

```diff
@@ -20,6 +20,7 @@ repos:
     hooks:
     -   id: autoflake
         args: ['--in-place', '--remove-all-unused-imports']
+        language_version: python3.9
```

Comment:
3.8 to maximize compatibility pls. we'd like to flag any language features that aren't included in our lowest supported version